Scripts/Extensions/TransformExtensions.cs

```diff
@@ -22,6 +22,24 @@ public static Bounds TransformBounds(this Transform transform, Bounds localBound
 
             return new Bounds { center = center, extents = extents };
         }
+
+        public static Vector3 XZForward(this Transform target)
+        {
+            var forward = target.forward;
+            if ((forward.y * forward.y) >= 0.5f)
+            {
+                forward = -target.up * Mathf.Sign(forward.y);
+            }
+            else
+            {
+                if (target.up.y < 0.0f)
```

Comment:
Convert to "else if" block, instead of an if block inside the else block:
...(else if (target.up.y < 0.0f)
{
    forward = -forward;
})